cdb2jdbc/src/main/java/com/bloomberg/comdb2/jdbc/Comdb2Connection.java

```diff
@@ -275,6 +273,10 @@ public void setStackAtOpen(boolean sendStack) {
         hndl.setSendStack(sendStack);
     }
 
+    public void setSkipResultSetDrain(boolean skipDrain) {
+        hndl.setSkipResultSetDrain(skipDrain);
+    }
+
```

Comment:
I think you need a setter for `clearAck` here, too.

Reply:
yep, it's there

Reply:
I don't see it in this patch. You need a setter in the Connection class.

Reply:
You're right. Method has been added now